src/components/document-picker.js

```diff
@@ -0,0 +1,86 @@
+import { Select } from 'antd';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import React, { useEffect, useState } from 'react';
+import { useService } from './container-context.js';
+import DocumentApiClient from '../api-clients/document-api-client.js';
+
+const { Option } = Select;
+
+function DocumentPicker({ documentId, onChange }) {
+  const { t } = useTranslation('documentPicker');
+  const documentApiClient = useService(DocumentApiClient);
```

Comment:
Maybe better `useSessionAwareApiClient` as the endpoint requires to be authenticated

Reply:
Yes of course, changed, thanks!